relays/ethereum-client/Cargo.toml

```
@@ -10,7 +10,9 @@ bp-eth-poa = { path = "../../primitives/ethereum-poa" }
 codec = { package = "parity-scale-codec", version = "2.0.0" }
 headers-relay = { path = "../headers-relay" }
 hex-literal = "0.3"
-jsonrpsee = { git = "https://github.com/svyatonik/jsonrpsee.git", branch = "shared-client-in-rpc-api", default-features = false, features = ["http"] }
+jsonrpsee-ws-client = { git = "https://github.com/paritytech/jsonrpsee" }
```

published alpha releases on crates.io that you can use https://crates.io/crates/jsonrpsee-ws-client
```suggestion
jsonrpsee-ws-client = "0.2.0-alpha"
```